src/operations/utils.c

```diff
@@ -15,4 +15,8 @@ void log_instruction_args(const champion_t *champion, const core_t *core, code_t
     i++;
   }
   printf("\n");
-}
\ No newline at end of file
+}
+
+int load_value_from_address(core_t *core, int address) {
```

Comment:
I can take care of this for you but for your future reference we have [vm read](https://github.com/yogimathius/core-wars/blob/additions/src/vm.c#L22) and [write methods](https://github.com/yogimathius/core-wars/blob/additions/src/vm.c#L31) to utilize for these operations 😁

I can update it though unless you have time! just lmk